src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridCell.cs

```diff
@@ -45,6 +48,9 @@ static DataGridCell()
 
             EventManager.RegisterClassHandler(typeof(DataGridCell), LostFocusEvent, new RoutedEventHandler(OnAnyLostFocus), true);
             EventManager.RegisterClassHandler(typeof(DataGridCell), GotFocusEvent, new RoutedEventHandler(OnAnyGotFocus), true);
+
```

Comment:
nit: two empty lines, if not intentional, you should probably leave only one.

Reply:
one empty line is added intentionally